server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/UpdateAction.java

```diff
@@ -65,52 +71,58 @@ public void define(NewController context) {
       .setResponseExample(getClass().getResource("example-update.json"))
```

Comment:
file `example-update.json` does not exist and has not been updated with new field `organizationKey\